### DISCUSSION TITLE
Украли XCH

### DISCUSSION THREAD

**Discussion by korvin970** [Chia-Network/chia-blockchain#15983] [Support]
good afternoon dear bot chia network. My CHIA wallet was robbed of all the XCH coins in there. the coins are still in the thief's wallet. I can prove the origin of these coins. The entire chain of acquisition and input to the wallet. I request you to investigate this matter. And if possible, return my tokens to me

**Comment by BrandtH22** [comment]
Hello @korvin970 , I am sorry to hear that the coins were stolen and urge you to review our article regarding how to be a hard target:
https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/ .

The chia blockchain is fully decentralized meaning we at Chia Network do not have any ability to return stolen or lost funds and are unable to assist in the recovery of funds.

**Comment by BrandtH22** [comment]
@korvin970 , unfortunately it appears someone has gained access to your keyset and/or device and stolen your funds. Since chia is a truly decentralized network there is no way to reclaim the funds or revert the transaction(s). One thing we recommend is ensuring that you and your farm are following security best practices to reduce malicious actors ability to gain access to your keys and/or device, many of these best practices are outlined here: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

Since we unfortunately cannot do more to assist with the stolen funds we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
